src/status_im2/subs/chat/chats.cljs

```diff
@@ -100,6 +100,13 @@
  (fn [{:keys [synced-from]}]
    synced-from))
 
+(re-frame/reg-sub
+ :chats/reaction-authors-list
+ (fn [[_ chat-id] _]
+   (re-frame/subscribe [:chat-by-id chat-id]))
+ (fn [{:message/keys [reaction-list]}]
```

Comment:
why are names different for the field and for the subscription?

Reply:
Renamed